pkg/controllers/provisioning/suite_test.go

```diff
@@ -259,6 +259,7 @@ var _ = Describe("Provisioning", func() {
 				},
 				Finalizers: []string{v1beta1.TerminationFinalizer},
 			},
+			Taints: []v1.Taint{v1beta1.UnregisteredNoExecuteTaint},
```

Comment:
Just curious: Why is this the only place in the scheduler code that requires the addition of this taint?